Possible missing header include on TOPPASVertexNameDialog.cpp & ui_TOPPASVertexNameDialog.h

Version : OpenMS 2.3
Platform : Ubuntu 14.04 64bit
Problem Occured when self-compiling

Error Message : 
make : TOPPASVertexNameDialog.cpp Error 
/home/enol/nDev/OpenMS/src/openms_gui/source/VISUAL/DIALOGS/TOPPASVertexNameDialog.cpp: In constructor ‘OpenMS::TOPPASVertexNameDialog::TOPPASVertexNameDialog(const QString&, const QString&)’:
/home/enol/nDev/OpenMS/src/openms_gui/source/VISUAL/DIALOGS/TOPPASVertexNameDialog.cpp:51:7: error: ‘QRegExpValidator’ was not declared in this scope
       QRegExpValidator* v = new QRegExpValidator(rx, ui_->line_edit);
       ^
/home/enol/nDev/OpenMS/src/openms_gui/source/VISUAL/DIALOGS/TOPPASVertexNameDialog.cpp:51:25: error: ‘v’ was not declared in this scope
       QRegExpValidator* v = new QRegExpValidator(rx, ui_->line_edit);
                         ^
/home/enol/nDev/OpenMS/src/openms_gui/source/VISUAL/DIALOGS/TOPPASVertexNameDialog.cpp:51:33: error: expected type-specifier before ‘QRegExpValidator’
       QRegExpValidator* v = new QRegExpValidator(rx, ui_->line_edit);
                                 ^
/home/enol/nDev/OpenMS/src/openms_gui/source/VISUAL/DIALOGS/TOPPASVertexNameDialog.cpp:51:33: error: expected ‘;’ before ‘QRegExpValidator’
make[2]: *** [src/openms_gui/CMakeFiles/OpenMS_GUI.dir/source/VISUAL/DIALOGS/TOPPASVertexNameDialog.cpp.o] Error 1 

Additional information :
I have handled this problem by manually adding 
'#include <QtGui/QRegExpValidator>'
at ui_TOPPASVertexNameDialog.h

If it's considered as a bug, and if this solution is applicable for the bug, please update the source for the better OpenMS!

--- a/src/openms_gui/source/VISUAL/DIALOGS/TOPPASVertexNameDialog.cpp
+++ b/src/openms_gui/source/VISUAL/DIALOGS/TOPPASVertexNameDialog.cpp
@@ -36,6 +36,8 @@
 #include <OpenMS/VISUAL/DIALOGS/TOPPASVertexNameDialog.h>
 #include <ui_TOPPASVertexNameDialog.h>
 
+#include <QRegExpValidator> 
+
 #include <iostream>
 
 namespace OpenMS

